### ISSUE TITLE
2.4.11 Focus Appearance exception doesn't match explanation

### ISSUE THREAD

**Issue by slauriat** [w3c/wcag#3016]
The user agent exception:

> The focus indicator is determined by the user agent and cannot be adjusted by the author, or

…implies that if the indicator is determined by the user agent and the author _can_ adjust it, the author must address any issue caused by the user agent.

Per [this response](https://github.com/w3c/wcag/issues/2706#issuecomment-1319397933), it sounds like the exception should read something more like:

> The focus indicator is determined by the user agent and has not been adjusted by the author, or

(opening a new issue per https://github.com/w3c/wcag/issues/2706#issuecomment-1422957968)

**Comment by mbgower** [comment]
There are two user agent exceptions:
> Exceptions:
The focus indicator is determined by the user agent and cannot be adjusted by the author, or
The focus indicator and the indicator's background color are not modified by the author.

These are explained in the Understanding document https://w3c.github.io/wcag/understanding/focus-appearance.html#exceptions Can you please review and clarify whether your questions still stand?

**Comment by mbgower** [comment]
PS I am currently working through the other issue you cite, trying to incorporate the response and varies comments into the Understanding document for greater clarity. I will cite this issue in the PR I create.

**Comment by slauriat** [comment]
We're definitely on the same page with Understanding documentation, but the SC itself has problematic wording that conflicts with that.

**Comment by alastc** [comment]
Hi @slauriat,

> Per https://github.com/w3c/wcag/issues/2706#issuecomment-1319397933, it sounds like the exception should read something more like:
> >  The focus indicator is determined by the user agent and has not been adjusted by the author, or

As mentioned in https://github.com/w3c/wcag/issues/2706#issuecomment-1319397933 there are scenarios where the author *cannot* adjust the focus indicator. That is a different scenario from the second user-agent exception, where the author can but has not.

**Comment by alastc** [comment]
Hi @slauriat,

I think I was confused about which UA exception you meant. I covered one above, the second one is for where an author changes something that affects the visibility. 

For example, a button on a plain HTML page, where the button has only `background-color: black` results in this in Edge (and I assume Chrome):
<img width="149" alt="two black buttons on a white page. One has slightly rounded corners." src="https://user-images.githubusercontent.com/216630/228586497-3040df14-d6cb-4eb6-9ecd-68c71b6fde68.png">

Not easy to distinguish the focused one...

It is phrased that way so that the author needs to check if they have adjusted the indicator, or it's background. However, I think the understanding doc needs updates for this, and due to recent updates many other things!

**Comment by slauriat** [comment]
Yep, I understand that. Another possible fix:

Exception:
- The focus indicator is determined by the user agent, and the indicator's background color is not modified by the author.

**Comment by alastc** [comment]
When we worked through the logic of applying this in HTML and PDF (and EPUB etc) we needed the clauses to be separate. I.e. in some scenarios authors can't change the indicator (but can change the background), and in other scenarios they control both. When we tried to combine them (as your suggestion does, I assume), there were undesirable outcomes.

For example, "determined" isn't enough to say whether the author can adjust it, or not. If they cannot adjust the indicator then it should meet an exception, e.g. in PDF.

If they can control it, but haven't adjusted it, that's the second exception.

We could not find a satisfactory way of combining them. 

Overall, this is still an issue with the understanding document, the SC text was carefully agreed previously and there is no new information here.

**Comment by slauriat** [comment]
See my suggestion in the description from when I opened the issue, which should cover that EPUB case while not implying that authors need to fix user agent issues:

> The focus indicator is determined by the user agent and has not been adjusted by the author, or

Some wording more consistent with other SCs that have to do with focus would work.

> Overall, this is still an issue with the understanding document…

No, I opened this on the SC language. We all seem to agree on the understanding document, so all good there. The currently drafted SC text still implies that authors need to fix user agent issues (I see the second exception, this just doesn't match the wording of other SCs like this and will introduce confusion if left unaddressed).

**Comment by alastc** [comment]
> See my suggestion in the description from when I opened the issue

I did, but I don't see how that covers the case of authors changing the background behind the indicator, such as the dark-button example above.

I'll put this in the survey to get a decision.

**Comment by slauriat** [comment]
If the user cannot alter the focus appearance, then they would not have altered it. The second exception in your comment also covers the first, so I think removing the first would avoid confusion.

I want to avoid any chance of interpretation that if the user _can_ alter the focus appearance, then they need to fix it even if it stems from the user agent, as that differs from the other SCs that note this kind of exception.

**Comment by mbgower** [comment]
>  The second exception in your comment also covers the first, so I think removing the first would avoid confusion.

I think that is true. We can still cover the situation where the user _can't_ change it in the Understanding document.

**Comment by alastc** [comment]
It doesn't cover the situation where the focus indicator is determined by the user agent, _can_ be modified by the author, and the background _is_ modified by the author.

That was a scenario we did want to cover, and is why they are separate.

Anyway, we [agreed in the meeting yesterday](https://www.w3.org/2023/05/02-ag-minutes#item06) to leave the SC text and tackle this in the understanding document. I'll leave this open until we've got a PR that covers this issue.

**Comment by bruce-usab** [comment]
Noting from Backlog conversation today (6/9/23) that the way HR now render in Understanding &mdash; [for Figure 10](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html#figure-focus-outer-green) the text below visually presents as belonging to Figure 11.  The next line after that _does_ belong to Figure 11 so the source order needs some tweaking regardless.
